src/lib/notification/index.ts

```diff
@@ -0,0 +1,9 @@
+// TODO -- this can just be imported in the required places as not gaining anything creating a new file for this
+import { Notify } from 'notiflix/build/notiflix-notify-aio';
+
+// export const Success = (message: string) => Notify.success(message);
+// export const Failure = (message: string) => Notify.failure(message);
+// export const Warning = (message: string) => Notify.warning(message);
+// export const Info = (message: string) => Notify.info(message);
```

Comment:
If you expect to use this code in the near future, please write a comment explaining what conditions need to be met to use these commented-out exports. Otherwise, let's just delete this entire file.

Often when we comment out code and don't explain when and why it should be restored, the commented-out code just hangs around forever as the next devs are afraid to delete it. So let's just go ahead and delete it now